pkg/images/knhandlers.go

```diff
@@ -0,0 +1,142 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package images
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/go-openapi/swag"
+	"github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+	"github.com/vmware/dispatch/pkg/images/backend"
+
+	dapi "github.com/vmware/dispatch/pkg/api/v1"
+	image "github.com/vmware/dispatch/pkg/images/gen/restapi/operations/image"
+	"github.com/vmware/dispatch/pkg/trace"
+	"github.com/vmware/dispatch/pkg/utils"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type knHandlers struct {
```

Comment:
There is nothing here that is knative specific outside of the backend.  I'm not sure that we need or should have a knative specific handler.

Reply:
I realize this is the pattern in place now, but we can change it.